pyaerocom/aeroval/config/ciconfigs/base_config.py

```diff
@@ -0,0 +1,182 @@
+"""
+this module contains basic aeroval configurations for running on CI
+"""
+
+import copy
+import logging
+from pathlib import Path
+
+from pyaerocom.io.pyaro.pyaro_config import PyaroConfig
+
+logger = logging.getLogger(__name__)
+
+# Constraints
+DEFAULT_RESAMPLE_CONSTRAINTS = dict(
+    yearly=dict(monthly=9),
+    monthly=dict(
+        daily=21,
+        weekly=3,
+    ),
+    daily=dict(hourly=18),
+)
+
+DEFAULT_RESAMPLE_CONSTRAINTS_DAILY = dict(
+    daily=dict(hourly=18),
+)
+
+HOMEDIR = Path.home()
+MYPYAEROCOM_DIR = Path.home() / "MyPyaerocom"
+MYPYAEROCOM_DIR.mkdir(exist_ok=True)
+TMP_DIR = Path.home() / "tmp"
+TMP_DIR.mkdir(exist_ok=True)
+JSON_DIR = Path.home() / "tmp" / "data"
+JSON_DIR.mkdir(exist_ok=True)
+COLDATA_DIR = Path.home() / "tmp" / "coldata"
+COLDATA_DIR.mkdir(exist_ok=True)
```

Comment:
This creates a bunch of files on import

Reply:
good point. Moved to the `get_CFG` method